.thoth.yaml

```diff
@@ -1,7 +1,6 @@
 host: khemenu.thoth-station.ninja
 
 managers:
-  - name: update
```

Comment:
You probably want to remove the whole manager `configuration`, i.e. the following 2 lines as well

Reply:
^^^